### PR TITLE
i18n SC-4821 Translation help

### PR DESCRIPTION
**Pull request by abralurrahman**
- [x] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-4821

### CODE REVIEW

File: views/help/lern-nuggets.hbs [hunk 1]
```diff
@@ -109,21 +109,21 @@
                         sowie digitalen Schulalltag kennen. Außerdem erfahrt ihr hier, was Open Educational
                         Ressources sind und wie ihr Lizenzen adäquat kennzeichnet.  <br>
```
Comment (dominikbruechner) [inline views/help/lern-nuggets.hbs:110]: missing i18n translation

File: views/help/lern-nuggets.hbs [hunk 2]
```diff
@@ -86,19 +86,19 @@
                         Was müsst ihr beim Einsatz an eurer Schule beachten? In dieser Lerneinheit stellen
```
Comment (dominikbruechner) [inline views/help/lern-nuggets.hbs:86]: missing i18n translation

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 59
           

Complexity decreasing per file
==============================
+ app.js  -1
+ controllers/dashboard.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 59
           

Complexity decreasing per file
==============================
+ controllers/dashboard.js  -2
+ app.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 59
           

Complexity decreasing per file
==============================
+ controllers/dashboard.js  -2
+ app.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by DevSchmidtchen** [comment]
Why did you change 170 files here? I think you changed some files back to the state without translation.

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 59
           

Complexity decreasing per file
==============================
+ helpers/recurringEvents.js  -1
+ controllers/dashboard.js  -2
+ app.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 59
           

Complexity decreasing per file
==============================
+ app.js  -1
+ helpers/recurringEvents.js  -1
+ controllers/dashboard.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by abralurrahman** [comment]
> Why did you change 170 files here? I think you changed some files back to the state without translation with your commit [b3c7111](https://github.com/schul-cloud/schulcloud-client/commit/b3c711178b01807ccebb3f705d97e8e47eea05ab).

I revert the merge cause lots of things are being changed which needed to revert.

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 59
           

Complexity decreasing per file
==============================
+ app.js  -1
+ controllers/dashboard.js  -2
+ helpers/recurringEvents.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Comment by dominikbruechner** [comment]
> > > Why did you change 170 files here? I think you changed some files back to the state without translation with your commit [b3c7111](https://github.com/schul-cloud/schulcloud-client/commit/b3c711178b01807ccebb3f705d97e8e47eea05ab).
> > 
> > 
> > I revert the merge cause lots of things are being changed which needed to revert.
> 
> Ok, but now there are in total 166 files changed and this can't be merged properly. Maybe you can renew your PR only with the files you really want to change here.

please do so @abralurrahman

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 60
           

Complexity decreasing per file
==============================
+ app.js  -1
+ controllers/dashboard.js  -2
+ helpers/recurringEvents.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5660824&bid=18530101)

**Pull request closed without merging**